docs/static/command-line-flags.asciidoc

```diff
@@ -63,6 +63,11 @@ Logstash has the following flags. You can use the `--help` flag to display this
 --reload-interval RELOAD_INTERVAL
   Specifies how often Logstash checks the config files for changes. The default is every 3 seconds.
 
+--pipeline.unsafe_shutdown
+  Force logstash to exit during shutdown even if there are still inflight events
+  in memory. By default, logstash will refuse to quit until all received events
+  have been pushed to the outputs.
+
 -h, --help
   Print help
 ----------------------------------
```

Comment:
Oh I will update that file with the web api options, thanks for bringing this to my attention.